code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -551,18 +551,21 @@
 	icon_icon = 'icons/mob/actions/actions_spells.dmi'
 	var/cooldown = 150
 	var/last_teleport = 0
+	var/is_charging = FALSE //Fixes bug 65900 preventing spam activation.
```

Comment:
Oh one thing.
Comments should reflect what the code does, rather then what it used to do.

Issue reports are also not always going to be linked to the git history (or even always still exist), so it's better to explain in a 
```
/// Doc comment
var/thing
```
about what the var exists to do explicitly

Reply:
Would you rather I recommit just removing the comment then? I can do that.

Reply:
make a new commit changing it to explain what the var is used for in more detail.
We squash commits, so it's not like it's gonna be in the git history forever,